src/components/button/code/templates/buttonAnchorTemplate.jsx

```diff
@@ -0,0 +1,16 @@
+var React = require('react');
+var classNames = require('classnames');
+var getComponentClasses = require('../../../../utils/getComponentClasses');
```

Comment:
I really don't like ../../../ etc.... can we not set a basepath ```global.__base = __dirname``` in ```src/ui-toolkit.js``` then can do ```require(__base + '/utils/getComponentClasses');``` <- double check my code ;)